sdk_container.sh

```diff
@@ -41,36 +45,43 @@ git checkout v3.9.1
 git submodule update --init --recursive
 ./autogen.sh
 ./configure
-make
+make -j $JOBS
 make check
 make install
 cd
 rm -rf protobuf
 
+# This makes sure that installed dynamic libraries are visible to the dynamic
+# linker, because it seems like make install does not take care of that
+ldconfig
+
 # emscripten
 git clone https://github.com/emscripten-core/emsdk.git
 cd emsdk
-./emsdk update-tags
-./emsdk install 3.1.7
-./emsdk activate 3.1.7
+git checkout 3.1.67
+./emsdk install --shallow 3.1.67
+./emsdk activate 3.1.67
 source ./emsdk_env.sh
 cd
 
+git clone https://github.com/protocolbuffers/protobuf protobuf-wasm
+cd protobuf-wasm
+git checkout v3.9.1
+git submodule update --init --recursive
+./autogen.sh
+emconfigure ./configure --disable-shared CXXFLAGS="-O3 -flto"
+emmake make -j $JOBS
+cd
+
+cp protobuf-wasm/src/.libs/libprotobuf-lite.a /sdk/libprotobuf-lite.a
+cp protobuf-wasm/src/.libs/libprotobuf.a /sdk/libprotobuf.a
```

Comment:
Since we're copying these files to /sdk/ now, I suspect we need to update the various Makefile references? (3 files to look at)

Reply:
We copied files to sdk before, the difference now is that copying happens inside sdk_container.sh and not in Dockerfile-sdk. That's because sdk_container.sh is now responsible for building it and we stop using prebuilt binaries. Everything else remains the same.

FWIW, I confirmed that Makefile.base and Makefile.base_lite work with the example from the docs (https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/main/docs/building.md), so I don't think that it's broken, at least not in an obvious way.

Reply:
Great! I missed that it was a noop. Do we still need this Makefile target?
https://github.com/proxy-wasm/proxy-wasm-cpp-sdk/blob/54167e2ba495ba603561f4560e527bda22fc9816/Makefile#L14-L25

Either way we can remove the wasm-patches bit.

Reply:
A good catch, I missed this one! I added a commit that fixes this Makefile as well, thank you!